jobs/ci-fejta-soak-deploy.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# Hack to test use a test image
+OVERRIDE="${WORKSPACE}/hack/jenkins/.kubekins_e2e_image_tag"
+mkdir -p $(dirname "${OVERRIDE}")
+echo v20161116-92cc67c > "${OVERRIDE}"
+
+readonly testinfra="$(dirname "${0}")/.."
+
+### provider-env
+export KUBERNETES_PROVIDER="gce"
+export E2E_MIN_STARTUP_PODS="8"
+export KUBE_GCE_ZONE="us-central1-f"
+export FAIL_ON_GCP_RESOURCE_LEAK="true"
+export CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS="1"
+export KUBE_NODE_OS_DISTRIBUTION="debian"
+
+### soak-env
+export FAIL_ON_GCP_RESOURCE_LEAK="false"
+export E2E_TEST="false"
+export E2E_DOWN="false"
+
+### job-env
+export PROJECT="fejta-prod"
+export JENKINS_SOAK_MODE="y"
+
+### post-env
+
+# Assume we're upping, testing, and downing a cluster
```

Comment:
Everything below here is boilerplate that doesn't vary between jobs.

This btw is exactly what jenkins does today :smile:

After we migrate everything over to bootstrap I plan to start changing a lot of this (using a configmap instead of bash to set environment variables, for example).